openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -82,6 +82,7 @@ def decom_packet(topic, _msg_id, msg_hash, _redis)
     # @param value [Object] The current value of the item
     # @param log_change [Boolean] Whether to log this limits change event
     def limits_change_callback(packet, item, old_limits_state, value, log_change)
+      return if @cancel_thread
```

Comment:
I added this during my early debugging but I think it's a small optimization that still makes sense